src/XMLimport.cpp

```diff
@@ -661,115 +666,34 @@ void XMLimport::readHelpPackage()
-        qDebug().nospace() << "XMLimport::readUnknownHostElement() ERROR: UNKNOWN "
-                              "Host Package Element, name: "
-                           << name().toString() << " and content: " << text().toString();
-
-        if (isEndElement()) {
-            break;
-        }
-    }
-}
-
-void XMLimport::readUnknownTriggerElement()
-{
-    while (!atEnd()) {
-        skipCurrentElement();
-        qDebug().nospace() << "XMLimport::readUnknownTriggerElement() ERROR: UNKNOWN "
-                              "Trigger Package Element, name: "
-                           << name().toString() << " and content: " << text().toString();
-
-        if (isEndElement()) {
-            break;
-        }
-    }
-}
-
-void XMLimport::readUnknownTimerElement()
-{
-    while (!atEnd()) {
-        skipCurrentElement();
-        qDebug().nospace() << "XMLimport::readUnknownTimerElement() ERROR: UNKNOWN "
-                              "Timer Package Element, name: "
-                           << name().toString() << " and content: " << text().toString();
-
-        if (isEndElement()) {
-            break;
-        }
-    }
-}
-
-void XMLimport::readUnknownAliasElement()
-{
-    while (!atEnd()) {
-        skipCurrentElement();
-        qDebug().nospace() << "XMLimport::readUnknownAliasElement() ERROR: UNKNOWN "
-                              "Alias Package Element, name: "
-                           << name().toString() << " and content: " << text().toString();
-
-        if (isEndElement()) {
-            break;
-        }
-    }
-}
-
-void XMLimport::readUnknownActionElement()
-{
-    while (!atEnd()) {
-        skipCurrentElement();
-        qDebug().nospace() << "XMLimport::readUnknownActionElement() ERROR: UNKNOWN "
-                              "Action Package Element, name: "
-                           << name().toString() << " and content: " << text().toString();
-
-        if (isEndElement()) {
-            break;
-        }
-    }
-}
-
-void XMLimport::readUnknownScriptElement()
-{
-    while (!atEnd()) {
-        skipCurrentElement();
-        qDebug().nospace() << "XMLimport::readUnknownScriptElement() ERROR: UNKNOWN "
-                              "Script Package Element, name: "
-                           << name().toString() << " and content: " << text().toString();
-
-        if (isEndElement()) {
-            break;
-        }
-    }
-}
-
-void XMLimport::readUnknownKeyElement()
-{
-    while (!atEnd()) {
-        skipCurrentElement();
-        qDebug().nospace() << "XMLimport::readUnknownKeyElement() ERROR: UNKNOWN "
-                              "Key Package Element, name: "
-                           << name().toString() << " and content: " << text().toString();
-
-        if (isEndElement()) {
-            break;
+    if (!atEnd()) {
+        qDebug().nospace().noquote() << "XMLimport::readUnknownElement(\"" << what << "\") ERROR - UNKNOWN Package Element name: \"" << name().toString() << "\".";
```

Comment:
While this is human-readable, you can't paste it easily in a code editor to go to the line and column right away. Suggest changing it to `line:column` syntax.

Reply:
Not to block that on such trivial thing to change later if neede I'll just merge it.